lab1PO/FigureList.cpp

```diff
@@ -0,0 +1,120 @@
+#include <iostream>
+#include "FigureList.h"
+#include "Kolo.h"
+#include "Prostokat.h"
+#include "Trojkat.h"
+using namespace std;
+
+
+FigureList::FigureList(double a, double b, double c, double r) {
```

Comment:
This constructor seems to be not natural. IMHO, there should be on argument with an array called initialFigures.